packages/state-transition/src/block/processAttestationPhase0.ts

```diff
@@ -100,15 +100,27 @@ export function validateAttestation(fork: ForkSeq, state: CachedBeaconStateAllFo
       );
     }
 
-    // Get total number of attestation participant of every committee specified
-    const participantCount = committeeIndices
-      .map((committeeIndex) => epochCtx.getBeaconCommittee(data.slot, committeeIndex).length)
-      .reduce((acc, committeeSize) => acc + committeeSize, 0);
+    const validatorsByCommittee = epochCtx.getBeaconCommittees(data.slot, committeeIndices);
+    const aggregationBitsArray = attestationElectra.aggregationBits.toBoolArray();
 
+    // Total number of attestation participant of every committee specified
+    let committeeOffset = 0;
+    for (const committee of validatorsByCommittee) {
+      const committeeAggregationBits = aggregationBitsArray.slice(committeeOffset, committee.length);
+
+      // Assert aggregation bits in this committee have at least one true bit
+      if (committeeAggregationBits.every((bit) => !bit)) {
+        throw new Error("Every committee in committee bits must have at least one attester");
```

Comment:
shouldn't this say
```suggestion
        throw new Error("Every committee in aggregation bits must have at least one attester");
```

otherwise might be confused with `committeeBits`